source/userguide/dm.rst

```diff
@@ -567,6 +567,96 @@ There are three ways to exit a Dialogue Flow:
 
 As shown here, you can use the dialog flow functionality to effectively craft complex flows that gracefully direct the user to provide the desired information for your application.
 
+.. _auto_slot_filling_for_entities:
+
+Automatic Slot Filling for Entities
+-----------------------------------
+
+MindMeld provides a useful functionality for automatically prompting the user for missing entities or slots required to fulfill an intent. This is done via applying the ``@app.auto_fill`` decorator for the dialogue state handler requiring entities to be obtained prior to applying the functionality defined within.
+
+This decorator replaces the need to define the ``@app.handle`` decorator. MindMeld will prompt the user for missing entities before applying the handler functionality by itself. The arguments required for this decorator are all the rules that would apply for that dialogue state (such as domain, intent, entities etc.), with the addition of a ``form`` argument.
+
+- ``form`` is a dictionary containing the following entries:
+
+  - ``entities`` (list, required): List of ``FormEntity`` objects with each defined for one entity slot to be filled.
+  - ``max_retries`` (int, optional, default 1): maximum number of retries allowed per entity or slot if user response is invalid.
+  - ``exit_msg`` (str, optional): If slot filling is exited abruptly without completion, define custom message to display.
+  - ``exit_intent`` (list, optional): If slot filling is exited abruptly without completion, the developer can define which particular intents (format ``['domain.intent', 'domain.intent'``) should the following query be redirected to. Default: None.
```

Comment:
Could we explain a bit further here that the format for exit intent is `domain.intent` or `domain.*`?

Reply:
Removed this feature after speaking with Minh, seems like it could add confusion.